### PR TITLE
Add instructions on how to use conda to run vvv tutorials and assignment

### PR DESCRIPTION
**Pull request by traversaro**
Most of this are due to @xela-95 . The official way of running the tutorials remains the Docker image, but as we tried to use conda and it was working fine, we thought it was worth to document it somewhere.

### CODE REVIEW

**Comment by traversaro** [comment]
@xela-95 I can't add you as a formal reviewer here, but please review the PR as well, thanks!

**Comment by pattacini** [comment]
> @xela-95 I can't add you as a formal reviewer here, but please review the PR as well, thanks!

I've invited @xela-95 as a member of this org.
Once accepted, we should be able to assign him as a reviewer.

File: instructions/how-to-create-conda-environment-for-school.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+export GAZEBO_MODEL_PATH=${CONDA_PREFIX}/share/gazebo/models:${GAZEBO_MODEL_PATH}
+export GAZEBO_RESOURCE_PATH=${CONDA_PREFIX}/share/gazebo/worlds:${GAZEBO_RESOURCE_PATH}
+~~~
+
+and adding it in the `${CONDA_PREFIX}/etc/conda/activate.d` directory.
+
+Then, whenever to run a command, remember to run it in the `vvvschool` conda environment.
+
+> [!WARNING]  
+> Not all vvv repos may not be working fine with conda. If you encounter problems, please open an issue in this repo, tagging @traversaro .
```
Comment (pattacini) [inline instructions/how-to-create-conda-environment-for-school.md]:
```suggestion
> Not all vvv-school repos may be working fine with conda. If you encounter problems, please open an issue in this repo, tagging @traversaro.
```

**Review by pattacini — CHANGES_REQUESTED** [review]
Thanks @traversaro for your contribution!
Very much appreciated!

Just an inline comment.

**Comment by xela-95** [comment]
> > @xela-95 I can't add you as a formal reviewer here, but please review the PR as well, thanks!
> 
> I've invited @xela-95 as a member of this org. Once accepted, we should be able to assign him as a reviewer.

Thank you! I've accepted the invitation.

**Comment by traversaro** [comment]
> Thanks @traversaro for your contribution! Very much appreciated!
> 
> Just an inline comment.

Fixed!

**Comment by pattacini** [comment]
> Thank you! I've accepted the invitation.

Unfortunately, being a member of the org is not sufficient to appear as a possible reviewer of this repo.
At any rate, go ahead with your review @xela-95!

**Review by pattacini — APPROVED** [review]

**Comment by traversaro** [comment]
> Unfortunately, being a member of the org is not sufficient to appear as a possible reviewer of this repo.

Yes, probably we would need to have him as a collaborator of the repo, at least with read permissions. Anyhow, it is not important, @xela-95 can do a review even if I do not add him explicitly as a reviewer.

**Comment by pattacini** [comment]
> @xela-95 can do a review even if I do not add him explicitly as a reviewer.

@xela-95, as soon as you give me your go here below, I'll merge the PR.

**Comment by xela-95** [comment]
> @xela-95, as soon as you give me your go here below, I'll merge the PR.

@pattacini Ok also for me it's fine! Approving :)

**Pull request merged**